.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: ruby
+rvm:
+  - 2.0
+  - 2.1
+  - 2.3
+  - 2.3
+  - 2.4
+  - 2.6
+  - 2.7
+before_install:
+   sudo service redis-server start --bind 127.0.0.1
```

Comment:
I think we need something like this instead:
```
services:
  - redis-server
```
See: https://docs.travis-ci.com/user/database-setup/#redis

Reply:
This worked and builds started passing. Should I change it anyway?